azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTest2.java

```diff
@@ -110,7 +115,9 @@ public void setUp() throws Exception {
     if (this.workingDir.exists()) {
       FileUtils.deleteDirectory(this.workingDir);
     }
-    this.workingDir.mkdirs();
+    if (!this.workingDir.mkdirs()) {
+      throw new RuntimeException("Failed to create working directory: " + this.workingDir);
+    }
```

Comment:
As `FileUtils` is already used on the previous "line", this could be replaced by a one-liner: `FileUtils.forceMkdir(this.workingDir);`